src/main/java/org/opensearch/security/dlic/rest/api/AccountApiAction.java

```diff
@@ -54,15 +54,15 @@
 import com.google.common.collect.ImmutableList;
 
 import static org.opensearch.security.dlic.rest.support.Utils.hash;
-import static org.opensearch.security.dlic.rest.support.Utils.addRoutesPrefix;
+import static org.opensearch.security.dlic.rest.support.Utils.replaceRoutes;
 
 /**
  * Rest API action to fetch or update account details of the signed-in user.
  * Currently this action serves GET and PUT request for /_opendistro/_security/api/account endpoint
  */
 public class AccountApiAction extends AbstractApiAction {
     private static final String RESOURCE_NAME = "account";
-    private static final List<Route> routes = addRoutesPrefix(ImmutableList.of(
+    private static final List<ReplacedRoute> replacedRoutes = replaceRoutes(ImmutableList.of(
```

Comment:
should we just keep the veriable as  routes?

Reply:
As we are replacing the old routes with the new routes in the current REST APIs, so we decided to choose a different name. For the new APIs we will use routes.